pkg/events/event.go

```diff
@@ -54,6 +54,7 @@ const (
 	ShowRestorePointsStarted        = "ShowRestorePointsStarted"
 	ShowRestorePointsFailed         = "ShowRestorePointsFailed"
 	RestoreNotSupported             = "RestoreNotSupported"
+	ReplicationNotSupported         = "ReplicationNotSupported"
```

Comment:
I noticed that we have been splitting the constants out by controllers. Some of the vrpq constants are in here too. Could you move them to separate blocks?